ledger.go

```diff
@@ -184,12 +185,10 @@ func NewLedger(kv store.KV, client *skademlia.Client, opts ...Option) (*Ledger,
 
 		sync: make(chan struct{}),
 
-		cacheCollapse: NewCollapseLRU(16),
-		fileBuffers:   filebuffer.NewPool(sys.SyncPooledFileSize, ""),
-
-		sendQuota: make(chan struct{}, 2000),
+		stateLRU: NewStateLRU(16),
+		filePool: filebuffer.NewPool(sys.SyncPooledFileSize, ""),
 
-		transactionsSyncIndex: cuckoo.NewFilter(),
+		filter: cuckoo.NewFilter(),
```

Comment:
previous naming was explaining what this structure is used for rather than just what this structure is, which can be determined by type btw.